EIPS/eip-draft-extclear.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: <to be assigned>
+title: EXTCLEAR Opcode For SELFDESTRUCTed contracts
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/eip-2936-extclear-for-selfdestruct/4569
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-03
+replaces: 2751
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+`EXTCLEAR` Opcode For `SELFDESTRUCT`ed contracts
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+Enable new opcode to clear storage for `SELFDESTRUCTED`ed contracts.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+`SELFDESTRUCT` complexity can be reduced by deferring state cleanup.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+`SELFDESTRUCT` is unnecessarily complex because it clears an unbounded amount of contract storage.
+It is computationally expensive for nodes to track all of the storage used in every contract in case the contract `SELFDESTRUCT`s.
+Further, contracts can be re-initialized using `CREATE2`, and then `SLOAD` prior storage.
+Therefore, several ethereum clients do not clear storage at all, and just check if the contract was initiated since `SSTORE` during `SLOAD`.
```

Comment:
```suggestion
Therefore, several ethereum clients do not clear storage at all, and just check if the contract was initiated since `SSTORE` (`0x55`) during `SLOAD`.
```